src/injector.ts

```diff
@@ -10,6 +10,16 @@ export type InjectOptions = {
 	 * Set this to true to include them instead.
 	 */
 	includeUnusedProps?: boolean;
+	/**
+	 * Given the `prop`, the `previous` source of the validator and the `generated` source:
+	 * What source should be injected?
+	 * @default Uses `generated` source
+	 */
+	reconcilePropTypes?: (
+		prop: t.PropTypeNode,
+		previous: string | undefined,
+		generated: string
+	) => string;
```

Comment:
Could you change this to get the type from `GenerateOptions`?
https://github.com/merceyz/typescript-to-proptypes/blob/ce9a7002c7fda27965b50e0b1af3ecef540a90e5/src/injector.ts#L34

Reply:
Done